docs/source/tutorial-tracking-ui.rst

```diff
@@ -0,0 +1,144 @@
+
+The runs in the **hyperparam** example diverge: the runs either generate metrics near ``0.9`` or near ``0.5``. You might want to explore only the better runs. In the search box, enter the following:
+
+.. code-block:: sql
+
+  metrics.val_rmse < 0.55
+
+Select the **Refresh** button. Hover your mouse over various lines to see a popup showing run details. Select the run with the lowest RMSE. Click the run name in the popup box. This will open the run's detail page.
+
+Search with SQL WHERE subset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A search filter is one or more expressions joined by the ``AND`` keyword. The search syntax does not support ``OR``. Each expression has three parts: an identifier of the target entity (for instance, ``metrics.accuracy``), a comparator (for instance, ``>=`` for numeric values, ``LIKE`` for strings), and a constant. For example:
+
+.. code-block:: sql
+
+  metrics.accuracy >= 0.9 AND params.model_name LIKE 'RESNET%'
+
+The target entities are:
+
+* ``metrics``: A metric logged with ``mlflow.log_metric``.
+* ``params``: A parameter logged with ``mlflow.log_param``.
+* ``tags``: A tag logged with ``mlflow.set_tag``.
+* ``attribute.run_id``: The ID of the run.
+* ``attribute.run_name``, ``run name``: The name of the run.
+* ``attribute.status``: The status of the run (``FINISHED``, ``FAILED``, ``RUNNING``, ``SCHEDULED``, ``KILLED``).
+* ``attribute.artifact_uri``: The URI of the artifact store.
+* ``attribute.user_id`` : The ID of the user who started the run.
+* ``attribute.start_time``, ``attribute.end_time`` : The start or end time of the run. Units are seconds elapsed in the Unix epoch (January 1, 1970). For example, ``start_time >= 1688169600`` filters runs created before 2023-07-01.
+
+For more, see :ref:`search-runs`.
+
+Run visibility
+~~~~~~~~~~~~~~~
+
+If, instead of defining a complete filter, you want to select among a handful of runs, you can toggle the visibility of runs in the filtered list. You do so with the column marked with an eye icon. Selecting the eye icon at the top of the column will allow you to toggle the visibility of all runs in the filtered list. Selecting the eye icon for an individual run will toggle the visibility of that run.
+
+Deleting runs
+~~~~~~~~~~~~~~
+
+You will have some runs that do not generate worthwhile results. Sometimes these will stem from code errors or other problems, but there are any number of reasons you may not want to keep a run.
+
+To delete a run, select it in the list of runs and select "Delete." You will be prompted to confirm the deletion. This puts the run in the ``Deleted`` state but *does not* remove the contents from the machine. The run is hidden by default, but can be searched for, viewed, and restored. To permanently remove the run and its associated data and artifacts, open a terminal and run:
+
+.. code-block:: bash
+
+  mlflow gc
+
```

Comment:
Can we add a note here that mentions that if the underlying artifact has been manually deleted (someone went into the file store and deleted the directory or files), a warning will be issued for the invalid location while gc continues to operate on the remaining elements that have been marked for deletion in the tracking server?